File: fixtures/rust_output/delete.rs
extern crate reqwest;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let res = reqwest::Client::new()
        .delete("http://www.url.com/page")
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}

File: fixtures/dart_output/delete.dart
import 'package:http/http.dart' as http;

void main() async {
  var res = await http.delete('http://www.url.com/page');
  if (res.statusCode != 200) throw Exception('delete error: statusCode= ${res.statusCode}');
  print(res.body);
}
